scripts/RunCombineFits.py

```diff
@@ -124,6 +125,9 @@ def RandomStringTag(size=6,chars=string.ascii_uppercase+string.ascii_lowercase+s
         CardNum = 1
         if args.ControlMode:
             TheFile = ROOT.TFile(os.environ['CMSSW_BASE']+"/src/auxiliaries/shapes/"+channel+"_controls_"+year+".root")
+        elif args.ComputeGOF:
+            TheFile = ROOT.TFile(os.environ['CMSSW_BASE']+"/src/auxiliaries/shapes/smh"+year+channel+"_GOF_nocorrelation.root")
```

Comment:
You need to run the card through the decorrelator (correlator) first. Calling it on the no correlation card will not work.